integrations/src/main/java/com/oms/dto/EmailRequestDto.java

```diff
@@ -1,49 +1,16 @@
-package com.oms.dto;
-
-public class EmailRequestDto {
-
-    private String salesOrderNumber;
-    private String messageTitle;
-    private String messageBody;
-    private String emailType;
-
-
-    public EmailRequestDto(String salesOrderNumber, String messageTitle, String messageBody, String emailType) {
```

Comment:
I don't understand why to remove this constructor - it also causes a compilation error since ModifyFulfillmentService is using it.